avaframe/com1DFA/DFAfunctionsCython.pyx

```diff
@@ -1991,3 +2037,116 @@ cpdef double SamosATfric(double rho, double Rs0, double mu, double kappa, double
   div = math.log(div) / kappa + B
   cdef double tau = p * mu * (1.0 + Rs0 / (Rs0 + Rs)) + rho * v * v / (div * div)
   return tau
+
+
+@cython.boundscheck(False)
+@cython.wraparound(False)
+@cython.cdivision(True)
+def computeIniMovement(cfg, particles, dem, dT, fields):
+  """ compute artificial viscosity only """
```

Comment:
- please add a docstring
- Should this be added to the particle source file?
- Please add a pytest for this.